source/_integrations/blinksticklight.markdown

```diff
@@ -29,12 +29,33 @@ light:
     serial: BS000795-1.1
 ```
 
+To add multiple LEDs from a multi-LED Blinkstick to your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+light:
+  - platform: blinksticklight
+    serial: BS000795-1.1
+    index: 0
+  - platform: blinksticklight
+    serial: BS000795-1.1
+    index: 1
+  - platform: blinksticklight
+    serial: BS000795-1.1
+    index: 2
+```
+
 {% configuration %}
 serial:
   description: The serial number of your stick.
   required: true
   default: 640
   type: string
+index:
```

Comment:
We no longer accept YAML configuration for integrations that connect to devices or services.

Instead, a configuration flow via the UI should be used.

For more information about this decision see:

- ADR-0007: <https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md#decision>
- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

To get started on the integration configuration, see our developer documentation:

<https://developers.home-assistant.io/docs/config_entries_config_flow_handler>